docs/DeveloperGuide.adoc

```diff
@@ -1120,7 +1133,7 @@ Use case ends
 
 *MSS*
 
-1. User keys in command to view the information and statistics of his weekly/monthly/yearly budget
+1. User keys in command to view the information and statistics of his weekly/monthly/yearly totalBudget
```

Comment:
totalBudget should be `totalBudget`.
Should also refer to it as just "totalBudget" instead of "the totalBudget" or "a totalBudget"